pkg/agent/controller/networkpolicy/reconciler.go

```diff
@@ -290,26 +315,67 @@ func (r *reconciler) Reconcile(rule *CompletedRule) error {
 	return ofRuleInstallErr
 }
 
+func (r *reconciler) getRuleType(rule *CompletedRule) ruleType {
+	if !r.multicastEnabled {
+		return unicast
+	}
+	for _, service := range rule.Services {
```

Comment:
Is there any validation webhook and doc that make sure a rule can either be all or none about IGMP?